CHANGES.rst

```diff
@@ -51,6 +51,8 @@ Pull requests
 - Function to plot Fermi 3FGL light curves [#286] (Jonathan Harris)
 - Add EffectiveAreaTable exporter to EffectiveAreaTable2D [#276] (Johannes King)
 - Add interface to HESS FitSpectrum JSON output [#296] (Christoph Deil)
+- Remove gammapy.shower package [#291] (Christoph Deil)
```

Comment:
I don't understand why the `gammapy.shower` line shows up in the diff for this PR!?
I added it here: https://github.com/gammapy/gammapy/pull/291/files
Now in master it doesn't seem to be there any more: https://github.com/gammapy/gammapy/blame/master/CHANGES.rst#L51

Well ... never mind ... if it's not there it's good it's being added back.